src/operator/nn/softmax-inl.h

```diff
@@ -207,8 +207,7 @@ struct SoftmaxParam : public dmlc::Parameter<SoftmaxParam> {
   int axis;
   DMLC_DECLARE_PARAMETER(SoftmaxParam) {
     DMLC_DECLARE_FIELD(axis).set_default(-1)
-      .describe("The axis along which to compute softmax. "
-                "By default use the last axis");
+      .describe("The axis along which to compute softmax.");
```

Comment:
default is mentioned in the parameters. don't see a need to call out, we are not calling out the defaults in other APIs